source/ftrack_connect_pipeline/plugin/load/importer.py

```diff
@@ -118,7 +118,7 @@ def load_asset(self, context_data=None, data=None, options=None):
         result = {
             'asset_info': self.asset_info,
             'dcc_object': self.dcc_object,
-            'run_method': run_result,
+            'run_result': run_result,
```

Comment:
Did you check what the run_result returned? 
Basically check if the result is just the plugin result or the result is something like:
{
run: "result of the run method",
fetch: "result of the fetch method",
}
If the result is this one, then run method is correct.

Reply:
Yes I checked this and run_result contained the raw output from definition importer plugin, eg. {"<component path>":"<imported path>"}

Reply:
So it contains the method? {"<method_name>":"<result>"} Then is better to name it run_method_result.